lib/autoload/kuroko2/workflow/engine.rb

```diff
@@ -72,6 +72,21 @@ def failure(token)
         end
       end
 
+      def auto_retry(node, token)
```

Comment:
ps: it's better to use just 9169e77, not `9169e77` to let github link commit automatically XD